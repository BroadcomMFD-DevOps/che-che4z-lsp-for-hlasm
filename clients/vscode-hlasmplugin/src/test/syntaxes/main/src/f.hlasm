         AIF   ( NOT ( (('A' LE '&C') AND ('&C' LE 'I')).AAAA OR       X        
                       (('J' LE '&C') AND ('&C' LE 'R'))  OR           X        
                       (('S' LE '&C') AND ('&C' LE 'Z'))  OR           X       
                                                                       X
                       ('&C' EQ '$') OR ('&C' EQ '_')     OR           X        
                       ('&C' EQ '#') OR ('&C' EQ '@')                  X        
                     )                                                 X        
               ).NAMERR         