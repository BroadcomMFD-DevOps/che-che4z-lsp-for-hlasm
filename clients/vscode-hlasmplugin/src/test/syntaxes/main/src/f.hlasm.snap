>         AIF   ( NOT ( (('A' LE '&C') AND ('&C' LE 'I')).AAAA OR       X        
#^^^^^^^^^^^^ source.hlasm entity.name.function.hlasm
#            ^^^ source.hlasm
#               ^^ source.hlasm
#                 ^^^ source.hlasm punctuation.definition.tag.hlasm
#                    ^ source.hlasm
#                     ^^ source.hlasm
#                       ^ source.hlasm
#                        ^ source.hlasm
#                         ^ source.hlasm string.hlasm
#                          ^^ source.hlasm string.hlasm
#                            ^ source.hlasm
#                             ^^ source.hlasm punctuation.definition.tag.hlasm
#                               ^ source.hlasm
#                                ^ source.hlasm string.hlasm
#                                 ^^ source.hlasm variable.other.readwrite.hlasm
#                                   ^ source.hlasm string.hlasm
#                                    ^ source.hlasm
#                                     ^ source.hlasm
#                                      ^^^ source.hlasm punctuation.definition.tag.hlasm
#                                         ^ source.hlasm
#                                          ^ source.hlasm
#                                           ^ source.hlasm string.hlasm
#                                            ^^ source.hlasm variable.other.readwrite.hlasm
#                                              ^ source.hlasm string.hlasm
#                                               ^ source.hlasm
#                                                ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                  ^ source.hlasm
#                                                   ^ source.hlasm string.hlasm
#                                                    ^^ source.hlasm string.hlasm
#                                                      ^ source.hlasm
#                                                       ^ source.hlasm
#                                                        ^ source.hlasm punctuation.definition.tag.hlasm keyword.operator.hlasm
#                                                         ^^^^ source.hlasm punctuation.definition.tag.hlasm
#                                                             ^ source.hlasm
#                                                              ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                                ^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>                       (('J' LE '&C') AND ('&C' LE 'R'))  OR           X        
#^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                       ^ source.hlasm
#                        ^ source.hlasm
#                         ^ source.hlasm string.hlasm
#                          ^^ source.hlasm string.hlasm
#                            ^ source.hlasm
#                             ^^ source.hlasm punctuation.definition.tag.hlasm
#                               ^ source.hlasm
#                                ^ source.hlasm string.hlasm
#                                 ^^ source.hlasm variable.other.readwrite.hlasm
#                                   ^ source.hlasm string.hlasm
#                                    ^ source.hlasm
#                                     ^ source.hlasm
#                                      ^^^ source.hlasm punctuation.definition.tag.hlasm
#                                         ^ source.hlasm
#                                          ^ source.hlasm
#                                           ^ source.hlasm string.hlasm
#                                            ^^ source.hlasm variable.other.readwrite.hlasm
#                                              ^ source.hlasm string.hlasm
#                                               ^ source.hlasm
#                                                ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                  ^ source.hlasm
#                                                   ^ source.hlasm string.hlasm
#                                                    ^^ source.hlasm string.hlasm
#                                                      ^ source.hlasm
#                                                       ^ source.hlasm
#                                                        ^^ source.hlasm
#                                                          ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                            ^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>                       (('S' LE '&C') AND ('&C' LE 'Z'))  OR           X       
#^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                       ^ source.hlasm
#                        ^ source.hlasm
#                         ^ source.hlasm string.hlasm
#                          ^^ source.hlasm string.hlasm
#                            ^ source.hlasm
#                             ^^ source.hlasm punctuation.definition.tag.hlasm
#                               ^ source.hlasm
#                                ^ source.hlasm string.hlasm
#                                 ^^ source.hlasm variable.other.readwrite.hlasm
#                                   ^ source.hlasm string.hlasm
#                                    ^ source.hlasm
#                                     ^ source.hlasm
#                                      ^^^ source.hlasm punctuation.definition.tag.hlasm
#                                         ^ source.hlasm
#                                          ^ source.hlasm
#                                           ^ source.hlasm string.hlasm
#                                            ^^ source.hlasm variable.other.readwrite.hlasm
#                                              ^ source.hlasm string.hlasm
#                                               ^ source.hlasm
#                                                ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                  ^ source.hlasm
#                                                   ^ source.hlasm string.hlasm
#                                                    ^^ source.hlasm string.hlasm
#                                                      ^ source.hlasm
#                                                       ^ source.hlasm
#                                                        ^^ source.hlasm
#                                                          ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                            ^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^ source.hlasm invalid.ignored
>                                                                       X
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
>                       ('&C' EQ '$') OR ('&C' EQ '_')     OR           X        
#^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                       ^ source.hlasm
#                        ^ source.hlasm string.hlasm
#                         ^^ source.hlasm variable.other.readwrite.hlasm
#                           ^ source.hlasm string.hlasm
#                            ^ source.hlasm
#                             ^^ source.hlasm punctuation.definition.tag.hlasm
#                               ^ source.hlasm
#                                ^ source.hlasm string.hlasm
#                                 ^^ source.hlasm string.hlasm
#                                   ^ source.hlasm
#                                    ^ source.hlasm
#                                     ^^ source.hlasm punctuation.definition.tag.hlasm
#                                       ^ source.hlasm
#                                        ^ source.hlasm
#                                         ^ source.hlasm string.hlasm
#                                          ^^ source.hlasm variable.other.readwrite.hlasm
#                                            ^ source.hlasm string.hlasm
#                                             ^ source.hlasm
#                                              ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                ^ source.hlasm
#                                                 ^ source.hlasm string.hlasm
#                                                  ^^ source.hlasm string.hlasm
#                                                    ^ source.hlasm
#                                                     ^^^^^ source.hlasm
#                                                          ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                            ^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>                       ('&C' EQ '#') OR ('&C' EQ '@')                  X        
#^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                       ^ source.hlasm
#                        ^ source.hlasm string.hlasm
#                         ^^ source.hlasm variable.other.readwrite.hlasm
#                           ^ source.hlasm string.hlasm
#                            ^ source.hlasm
#                             ^^ source.hlasm punctuation.definition.tag.hlasm
#                               ^ source.hlasm
#                                ^ source.hlasm string.hlasm
#                                 ^^ source.hlasm string.hlasm
#                                   ^ source.hlasm
#                                    ^ source.hlasm
#                                     ^^ source.hlasm punctuation.definition.tag.hlasm
#                                       ^ source.hlasm
#                                        ^ source.hlasm
#                                         ^ source.hlasm string.hlasm
#                                          ^^ source.hlasm variable.other.readwrite.hlasm
#                                            ^ source.hlasm string.hlasm
#                                             ^ source.hlasm
#                                              ^^ source.hlasm punctuation.definition.tag.hlasm
#                                                ^ source.hlasm
#                                                 ^ source.hlasm string.hlasm
#                                                  ^^ source.hlasm string.hlasm
#                                                    ^ source.hlasm
#                                                     ^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>                     )                                                 X        
#^^^^^^^^^^^^^^^^^^^^^ source.hlasm
#                     ^ source.hlasm
#                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasm keyword.other.hlasm
#                                                                        ^^^^^^^^ source.hlasm invalid.ignored
>               ).NAMERR         
#^^^^^^^^^^^^^^^ source.hlasm
#               ^ source.hlasm
#                ^^^^^^^ source.hlasm keyword.control.hlasm
#                       ^^^^^^^^^ source.hlasm